build-in-container/scripts/setup.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+echo "------------ Setting up Mariner Build Environment ------------"
+
+# build variables
+BUILD_DIR="/tmp/mariner/build"
```

Comment:
Why do we want to re-direct these into `/tmp`? I would expect we would want to preserve the normal layout of the repo wouldn't we? (ie, I'd expect `cd ../build` to take me to my build directory both in the container and outside the container).